Fix Qt version detection (we don't need libproxy with Qt >=5.5.0)

Index: src/remotepluginprocess/remotepluginprocess.pro
--- src/remotepluginprocess/remotepluginprocess.pro.orig
+++ src/remotepluginprocess/remotepluginprocess.pro
@@ -25,7 +25,7 @@ INCLUDEPATH += . \
 CONFIG += \
     build_all
 
-lessThan(QT_VERSION, "5.5.0"):system($$pkgConfigExecutable() --exists libproxy-1.0) {
+!versionAtLeast(QT_VERSION, "5.5.0"):system($$pkgConfigExecutable() --exists libproxy-1.0) {
     DEFINES += HAVE_LIBPROXY
     PKGCONFIG += libproxy-1.0
     SOURCES += my-network-proxy-factory.cpp
